Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -47,6 +47,9 @@ namespace Hazel {
 			glfwSetErrorCallback(GLFWErrorCallback);
 			s_GLFWInitialized = true;
 		}
+		
+		m_PrimaryMonitor = glfwGetPrimaryMonitor();
+		m_BaseVideoMode = *(glfwGetVideoMode(m_PrimaryMonitor));
 
 		m_Window = glfwCreateWindow((int)props.Width, (int)props.Height, m_Data.Title.c_str(), nullptr, nullptr);
```

Comment:
Shouldn't we directly pass the monitor to the create function in the case where we want to create fullscreen mode?
It feels wrong to create a windowed mode and then set it fullscreen afterward.

Note: calling `glfwSetWindowMonitor` to fullscreen inside `SetWindowMode` on an already fullscreen window has no effect. This might be more efficient than creating it windowed and changing to fullscreen.